src/utils/helpers/compareDates.js

```diff
@@ -0,0 +1,9 @@
+import { MILLI_SECONDS_IN_A_DAY } from "@utils/constants/values";
+
+export const compareDates = (startDate, endDate) => {
```

Comment:
what if there is no start date or end date? Maybe we can add some checking here.